packages/openapi-typescript/examples/github-api-next.ts

```diff
@@ -90333,15 +90335,97 @@ export interface operations {
     };
     requestBody: {
       content: {
-        "application/json": OneOf<[{
+        "application/json": ({
```

Comment:
This is a big improvement, and a scenario where maybe we should start preferring practice over theory